2018/python/test_file_generator.py

```diff
@@ -11,7 +11,14 @@ class test_data(object):
 	edits as well."""
 
 	def __init__(self, ts_schema, lar_schema, crosswalk_data):
-		"""Set initial class variables"""
+		"""Set initial class variables.
+
+		The crosswak_data variable contains the filepath and name for geographic 
```

Comment:
Can you put the filename in the docstring? I see several geo files in the dependency folder

Reply:
Sure! This change is reflected in the last commit.